src/pynxtools/dataconverter/convert.py

```diff
@@ -284,28 +284,32 @@ def main_cli():
     default=[],
     multiple=True,
     help=(
-        "Deprecated: Please use the positional file arguments instead. "
-        "The path to the input data file to read. (Repeat for more than one file.)"
+        "Deprecated: Please use the positional file arguments instead. The path to the "
+        "input data file to read. Repeat for more than one file. default=[] This option "
+        "is required if no '--params-file' is supplied."
     ),
 )
 @click.option(
     "--reader",
-    default="json_map",
+    default=None,
     type=click.Choice(get_names_of_all_readers(), case_sensitive=False),
-    help='The reader to use. default="example"',
+    help=(
+        "The reader to use. Examples are json_map or readers from a pynxtools plugin. "
+        "default=None This option is required if no '--params-file' is supplied."
+    ),
 )
 @click.option(
     "--nxdl",
     default=None,
     help=(
-        "The name of the NXDL file to use without extension."
-        "This option is required if no '--params-file' is supplied."
+        "The name of the NeXus application definition NXDL file to use without the "
+        "extension nxdl.xml. This option is required if no '--params-file' is supplied."
     ),
 )
 @click.option(
     "--output",
     default="output.nxs",
-    help="The path to the output NeXus file to be generated.",
+    help="The path to the output NeXus/HDF5 file to be generated.",
```

Comment:
Unnecessary tbh.

```suggestion
    help="The path to the output NeXus file to be generated.",
```

Reply:
No, NeXus is not required to write to HDF5 only

Reply:
NeXus file is an umbrella term but that implementation here writes an HDF5 file

Reply:
It's just semantic tbh. Isn't a HDF5 Nexus file a subset of a Nexus file? I don't mind this going either way. It's just we can explain how we only write HDF5 somewhere else if we need to. This seems to take space for no real added benefit. But again. I don't have a strong opinion on this. I just like to keep it as clean as possible. I'll let @lukaspie give his opinion/last decision on the matter?

Reply:
It really is just semantics, let's go with what is proposed here.